index.html

```diff
@@ -239,9 +238,8 @@ <h2 class="desktop">Uber</h2>
 
```

Comment:
### NOTICE: I kindly recommend you fix the above changes before merging your pull request as most of them were missed by the previous reviewer. I know you can do that :muscle: :muscle: